python/cugraph/cugraph/tests/test_compat_pr.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Temporarily suppress warnings till networkX fixes deprecation warnings
+# (Using or importing the ABCs from 'collections' instead of from
+# 'collections.abc' is deprecated, and in 3.8 it will stop working) for
+# python 3.7.  Also, this import networkx needs to be relocated in the
+# third-party group once this gets fixed.
+import pytest
+from cugraph.tests import utils
+import numpy as np
+import gc
+import importlib
+
+
+MAX_ITERATIONS = [100, 200]
+TOLERANCE = [1.0e-06]
+ALPHA = [0.85, 0.70]
+PERS_PERCENT = [0, 15]
+HAS_GUESS = [0, 1]
+
+FILES_UNDIRECTED = [
+    utils.RAPIDS_DATASET_ROOT_DIR_PATH/"karate.csv"
+]
+
+# these are only used in the missing parameter tests.
+KARATE_RANKING = [11, 9, 14, 15, 18, 20, 22,
+                  17, 21, 12, 26, 16, 28, 19]
+
+KARATE_PERS_RANKING = [11, 16, 17, 21, 4, 10, 5,
+                       6, 12, 7, 9, 24, 19, 25]
+
+KARATE_ITER_RANKINGS = [11, 9, 14, 15, 18, 20,
+                        22, 17, 21, 12, 26, 16,
+                        28, 19]
+
+KARATE_NSTART_RANKINGS = [11, 9, 14, 15, 18, 20,
+                          22, 17, 21, 12, 26, 16,
+                          28, 19]
+
+
+# =============================================================================
+# Pytest fixtures
+# =============================================================================
+def setup_function():
+    gc.collect()
+
+
+datasets = FILES_UNDIRECTED
+fixture_params = utils.genFixtureParamsProduct((datasets, "graph_file"),
+                                               (MAX_ITERATIONS, "max_iter"),
+                                               (TOLERANCE, "tol"),
+                                               (PERS_PERCENT, "pers_percent"),
+                                               (HAS_GUESS, "has_guess"),
+                                               )
+
+
+@pytest.fixture(scope="module", params=fixture_params)
+def input_combo(request):
+    """
+    Simply return the current combination of params as a dictionary for use in
+    tests or other parameterized fixtures.
+    """
+    print("parameters are \n", request.param, flush=True)
```

Comment:
another print statement here. We usually only print mismatches, modules version and benchmark numbers.